api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -67,4 +67,17 @@ message BandwidthLimit {
   // Runtime flag that controls whether the filter is enabled or not. If not specified, defaults
   // to enabled.
   config.core.v3.RuntimeFeatureFlag runtime_enabled = 5;
+
+  // Optional flag to enabled response trailers.
+  // Default is false.
+  bool enable_response_trailer = 6;
+
+  // Optional the prefix for the response trailers of bandwidth decode/encode delays for stream transfer.
```

Comment:
I don't really understand what this feature is or how it works from the API docs. Can you either expand them or add some explanation in the main Envoy docs and link there?

Reply:
Move to the description to `enable_response_trailers` where seems better place for the explanation.